### ISSUE TITLE
Store the name of the attachment file if present

### ISSUE THREAD

**Issue by sudar** [sudar/email-log#106]
Requested through support on Sep 13th.

The following should be done

- [x] Increment the DB Schema version
- [x] Add the following fields
- attachment_name - varchar (1000) - to store comma seperated list of attachment file names
- ip_address - varchar(15) - to store the ip address of the request (see #112)
- result - bool - to store the return value of `wp_mail` function (see #113)
- Increase the size of To field (see #31 and #116)
- [x] If the DB Schema version is old, then upgrade the schema
- [x] Capture the attachment file names and then store them with comma seperated values.

After that complete #112, #113 and #31

**Comment by mariadanieldeepak** [comment]
@sudar,

I'm able to store the attachment names.

![image](https://user-images.githubusercontent.com/17510033/44306146-3f7f3500-a3a6-11e8-8203-398cea761fa9.png)

When there is one attachment, a string value will be saved.

```
/tmp/0hathway-receipt.pdf
```

When there are more than one attachment, a comma separated string value will be saved.

```
/tmp/0subtle_dots.png,/tmp/1subtle_dots_@2X.png
```

**Comment by mariadanieldeepak** [comment]
@sudar,

This is now ready for your review at https://github.com/sudar/email-log/pull/155

**Comment by mariadanieldeepak** [comment]
@sudar,

This is now ready for your review at https://github.com/sudar/email-log/pull/155

**Comment by mariadanieldeepak** [comment]
- [ ] Find out to send emails from WP with attachments
    - [ ] Find Plugins and send email
- [ ] Find if you can read the file using the file path

**Comment by mariadanieldeepak** [comment]
List of plugins that sends email with attachments from WordPress

- [ ] https://wordpress.org/plugins/quick-mail/

**Comment by mariadanieldeepak** [comment]
@sudar,

I've fixed the following review comments that we discussed earlier.

![image](https://user-images.githubusercontent.com/17510033/44743653-b3e17300-ab20-11e8-898e-a1283cc09a62.png)

**Comment by mariadanieldeepak** [comment]
@sudar,

As discussed earlier, I've also created a PoC to see if we can retrieve the attachment file information. 
Based on the PoC, it is possible to retrieve information on attachments.

![image](https://user-images.githubusercontent.com/17510033/44743710-e12e2100-ab20-11e8-8039-bac89e5e372a.png)

The above screenshot shows the output of the following code.

https://github.com/sudar/email-log/commit/31706834afc8fe3540ad6e8b69279f83c3017a91

So far, I'm only able to find out only one plugin that sends attachment from WordPress and I've listed that out [here](https://github.com/sudar/email-log/issues/106#issuecomment-415323088).
